firebase.json

```diff
@@ -84,6 +84,7 @@
       { "source": "/sdk-archive", "destination": "/release/archive", "type": 301 },
       { "source": "/setup-:os*", "destination": "/get-started/install/:os*", "type": 301 },
       { "source": "/setup", "destination": "/get-started/install", "type": 301 },
+      { "source": "/spm", "destination": "/packages-and-plugins/swift-package-manager", "type": 301 },
```

Comment:
What will this be used for? I'd prefer to use a `/to/spm` [tooling redirect](https://github.com/flutter/flutter/blob/master/docs/contributing/use-reliable-links.md) if possible.

Reply:
It'll be used in a YouTube Short video. Good to use whichever format you prefer!